src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -26,12 +26,6 @@ internal static class NativeMethods
 #if NET45 || DNX451 || DNXCORE50
         static NativeMethods()
         {
-#if DNXCORE50
-            if(!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                return;
-            }
-#endif
```

Comment:
I think this code should stay. When DNX finishes implementing the runtime artifacts feature, it should start working correctly. The code in `NativeLibraryLoader` for `dnx451` & `dnxcore50` will also be removed when that happens.